data/mods/No_NPC_Needs/effects_on_condition.json

```diff
@@ -0,0 +1,15 @@
+[
+  {
+    "type": "effect_on_condition",
+    "id": "reset_needs",
+    "recurrence": "4 hours",
```

Comment:
So this means during the 4 hour interval the NPCs' needs will be calculated as usual right? If so this might not work in some extreme conditions such as overheating where you can become thirsty pretty quick I think.

Reply:
Correct, and good call on extreme conditions like that

Reply:
Testing shows that NPCs don't actually care about temperature, and thus don't sweat when it's 50 C out ¯\_(ツ)_/¯

Reply:
Can EOCs be triggered when the needs change, so that you can reset hunger, thirst etc once they fall below a certain value?